apps/mobile/src/components/Toast/Toast.tsx

```diff
@@ -67,7 +74,7 @@ export function AnimatedToast({ message, onClose = () => {}, autoClose = true, c
       style={[
         {
           // 56 is the height of the bottom navigation bar
-          bottom: bottom + 56,
+          bottom: bottom + (withoutNavbar ? 0 : 56),
```

Comment:
Adding `withoutNavbar` props since some of the screen have `navbar` and `without navbar`

Its for toast positioning